### DISCUSSION TITLE
Received an old M1, now what...

### DISCUSSION THREAD

**Discussion by pfuller88** [GerryDazoo/Slinger#350] [Q&A]
I got an M1 for $15.    I do not have the password.  I have it hooked up to my network and have used pfsense to block all incomming connections.  I can see it in slingbox PC software 1.5 - user name and finderID.   What are my options with this box.  If I reset it, it sounds like I can get slingbox 1.5 software to reset the admin password, but there is no way to map a guide or cablebox?  Is this correct?    Will it play one channel?

Any assistance is appreciated.

**Comment by GerryDazoo** [comment]
Without the current password you have no choice but to factory reset the box to reset the admin password to "admin". You should be able to stream from the box but the integrated IR blaster will not work (yet, we're working on it, don't give up hope).

**Reply by pfuller88** [reply]
thanks, I nothing but faith in you guys, great job.

**Reply by flipper2222222** [reply]
> Without the current password you have no choice but to factory reset the box to reset the admin password to "admin". You should be able to stream from the box but the integrated IR blaster will not work (yet, we're working on it, don't give up hope).

can you please confirm this would be the same case for a  non bricked Slingbox Pro  that I do not have the password for?

thanks, and many more thanks for your time investment in this ongoing project.

**Reply by GerryDazoo** [reply]
No that is not the case. We have a procedure now to totally unbrick the ProHD with the 1.5 software. Unfortunately it won't configure the 350/500/M1

**Reply by flipper2222222** [reply]
thanks  Gerry, I will go ahead and purchase it.

**Comment by pfuller88** [comment]
I tried reseting, it does not change the password to admin.   I looked at a bunch of the files in the sling media folder, one xml file has a long number which might be the  E1 number, would someone be able to run this through the decoder to see what comes back?

903AFC54850E2147A94D7CA5908704BD

**Reply by GerryDazoo** [reply]
You have to do a factory reset. Not just a reset. Hold the button down for 30 seconds.

I did and it fails miserably. Please post the complete line from the log file including the E1:
What you posted might be the FinderID.

**Comment by pfuller88** [comment]
I have tried reseting 4 or 5 times. I've held down the reset button for 60 seconds.  Still comes back with the old user. pic...
![image](https://user-images.githubusercontent.com/117112102/203863826-cb09ddab-8f9b-4b0d-b4aa-4ce582850caf.png)

Here is the xml file info...

`<?xml version="1.0" encoding="utf-8"?>
<SBProfile xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
	<DeviceDB>
		<Device dbid="{BED2DA53-C233-4B63-982E-915E219238DF}" username="guest" password="E1:" name="My Slingbox" port="5001" address="" useaddress="0" last.address="" last.port="0" id="903AFC54850E2147A94D7CA5908704BD"/></DeviceDB>
</SBProfile>`

**Comment by pfuller88** [comment]
as a side note, I tried to SSH into the slingbox.  I was able to connect, admin was not the password.  I tried as root user.  screenshot...

![image](https://user-images.githubusercontent.com/117112102/203867517-660d96ed-283a-48f4-8482-21da4bfc095c.png)

**Reply by GerryDazoo** [reply]
I was unaware that you could ssh into a slingbox.

**Reply by pfuller88** [reply]
I'll see if I can get in.  dropbear 2012 is quite old.

**Reply by sp222222** [reply]
> I was unaware that you could ssh into a slingbox.

ssh ...interesting. maybe someone can run a brute force on password

**Reply by pfuller88** [reply]
I've been trying brute force for the past few days, using Hydra and Crunch.  Nothing yet, have run through over 500,000 probes so far.

**Comment by GerryDazoo** [comment]
look for the E1: password in the console.log file. Use slingbox server to connect with password admin and post the logs if it doesn't work.

**Reply by Toei79** [reply]
where you locate console.log? thanks

i found it.

**Comment by rein-10** [comment]
i also have trouble resetting an m1, tried different ways of resetting with no luck

**Reply by RandomMan719** [reply]
@rein-10 
I have reset an m1 and an m2, before Nov.9th 2022.  I don't remember having issues with resetting, I have had issues resetting a ProHD, if the power supply is weak.  **_You might not be able to setup sources and or a remote control, if reset, hard reset at your own risk!!!_**  The link below might help you reset it.  

https://www.techsolutions.support.com/how-to/how-to-reset-a-slingbox-11862

**Comment by RocketAlf** [comment]
@pfuller88: In the meantime, is there any result for accessing the SSH interface?

**Reply by pfuller88** [reply]
no luck, tried brute force for a few days, nothing

**Comment by pfuller88** [comment]
I was able to reset the M1.  I held the reset for about 40 seconds to hard reset.  I then ran slingbox 5 software with no internet block. as it needed to login and the software updated the M1 firmware.  i then loaded a custom bin for the remote.  the software then took me back to the login page where it halted, although all was setup on the M1 and slinger works with admin password.

**Reply by almoney** [reply]
Which slingbox 5 software are you referring to here 5.0.0.83?

**Reply by pfuller88** [reply]
SlingplayerDesktop-5.0.0.83

**Reply by mike-uk** [reply]
@pfuller88 No internet block? Does it include port5678? doesn't it brick the box right away? How do you enter the software without logging in first? May be you did it while the log in server was still online.

**Reply by pfuller88** [reply]
Hi mike-uk.  Yes, no internt block was needed to use 5.00.83 Slingbox software setup.  After a hard reset of the M1,  the software updated my slingbox before allowing me to proceed.   I did need to log in to the slingbox software.   I reset my M1 just a few days ago using this process.   To note, I had torun the software  several times before the slingbox software completely finished and took me back to the login screen.   I then quit the software,  blocked the slingbox IP from the internet  and rebooted the slingbox.  It worked for me.

**Reply by mike-uk** [reply]
I see I have blocked the port 5678 for the while lan, I wonder why the sign in dont work. Thanks

If you just reset it with out going into sling account would it still work apart from not able to load the remote bin and change password?

**Reply by pfuller88** [reply]
I removed all my router blocks to the slingbox for this to work for my M1.  I put the router blocks back on after completed.

**Reply by pfuller88** [reply]
The only way I got it to work was as I describe.  The process does not prompt to change any password, the password of the slingbox is auto set to "admin" afterward.

**Reply by mike-uk** [reply]
@pfuller88 I just went to the second hand shop and bought a M1. Followed what you said, it did all the updates etc but I can't get pass the remote setup, after I confirm the remote is working. Not sure is it because I don't actually have an ir blaster plug in, as its missing. It just hang, close up or logout. However I install slingplayer2 and was able to see the the name is updated and I see what port it is on. I update slinger config.ini and it was working. So it is a bit odd but does work. Thank you very much!

**Reply by pfuller88** [reply]
Great.  To note, mine hung after the remote setup several times.   The 3rd time it finished.  Not sure why.  If you have it working, great.

**Reply by mike-uk** [reply]
So it can get through the remote setup if keep trying? I don't need one yet as I am waiting for my new set top box to come but its good to know. Thanks

Does the M1 really can get brick? It didn't get brick at all even with no blocks. But when I forget to restarting router after setting the block back and pluged my pro-hd back and that got brick right away.
